lib/einhorn/worker.rb

```diff
@@ -139,6 +134,33 @@ def self.graceful_shutdown(&blk)
 
     private
 
+    def self.handle_command_socket(discovery, contextual_arg)
```

Comment:
Err, either `contextual_arg` should be `arg` or you're missing some find-replace below, right?

Reply:
Nice, I guess no test exercises this :(

Fixed